singleton-ruby/LICENSE.txt

```diff
@@ -0,0 +1,283 @@
+Singleton
+
+Copyright (c) 2019 VMware, Inc.				

```

Comment:
Change 2019-> 2021

Reply:
removed it as it's unnecessary in Github.